test/modules/gs1-importer.test.js

```diff
@@ -151,6 +153,65 @@ describe('GS1 Importer tests', () => {
         });
     });
 
+    describe('Random vertices content check', async () => {
+        async function checkSpecificVerticeContent(xml) {
```

Comment:
Probably best would be to split each case into separate function. Also, test should fail in case any new file.